AsyncDisplayKit/Details/Transactions/_ASAsyncTransactionContainer.m

```diff
@@ -85,12 +81,10 @@ - (void)asyncdisplaykit_cancelAsyncTransactions
 
 - (void)asyncdisplaykit_asyncTransactionContainerStateDidChange
 {
-#if ASASYNCTRANSACTIONCONTAINER_FORWARD_STATE_CHANGE
   id delegate = self.delegate;
   if ([delegate respondsToSelector:@selector(asyncdisplaykit_asyncTransactionContainerStateDidChange)]) {
```

Comment:
@Adlai-Holler I had a few moments to look at this, and as I'm sure you were seeing originally, the integration point isn't that deep  (I had incorrectly assumed it was).

Let's go ahead and remove this functionality entirely (either in this diff or another).  We need to make other improvements to the transaction system and I suspect it may return, but it might as well be part of a larger audit.